charts/ssi-credential-issuer/Chart.yaml

```
@@ -20,8 +20,8 @@
 apiVersion: v2
 name: ssi-credential-issuer
 type: application
-version: 1.3.0
-appVersion: 1.3.0
+version: 1.3.1
+appVersion: 1.3.1
```

we'd release the fix together with the version for R25.06, or is the fix urgently needed?

```suggestion
version: 1.3.0
appVersion: 1.3.0
```